src/main/java/de/tum/in/www1/artemis/service/connectors/bamboo/dto/TestwiseCoverageReportDTO.java

```diff
@@ -0,0 +1,107 @@
+package de.tum.in.www1.artemis.service.connectors.bamboo.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class TestwiseCoverageReportDTO {
+
+    @JsonProperty("uniformPath")
+    private String uniformPath;
+
+    @JsonProperty("duration")
+    private double duration;
+
+    @JsonProperty("content")
+    private String content;
+
+    @JsonProperty("paths")
+    private List<CoveredPathsPerTestDTO> coveredPathsPerTestDTOS = new ArrayList<>();
```

Comment:
```suggestion
    private List<CoveredPathsPerTestDTO> coveredPathsPerTestDTOs = new ArrayList<>();
```
Pretty sure the s should be lowercase